application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -142,4 +146,35 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
                             + " the original channel you wanted it to be send to)");
         }
     }
-}
+
+    private void reminderFailureAction(int id, long channelId, long authorId,
+                                       CharSequence content, long guildId, int reminderFailureCounter) {
+        if (reminderFailureCounter <= 3) {
+            Instant current = Instant.now();
```

Comment:
instead of putting current here and computing the remindAt at the bottom, compute it up here. `Instant remindAt = Instant.now().plusMinutes(1)`